cognite/client/utils/__init__.py

```diff
@@ -14,3 +16,11 @@
     ms_to_datetime,
     timestamp_to_ms,
 )
+
+
+# Needed for doctest to pass.
+with contextlib.suppress(ImportError):
+    if sys.version_info >= (3, 9):
+        from zoneinfo import ZoneInfo
+    else:
+        from backports.zoneinfo import ZoneInfo
```

Comment:
went with the import_zoneinfo for clarity. But it is a good idea to introduce the function as it will replacing all these imports easier when we go to 3.9